soccer/gameplay/tactics/positions/wing_defender.py

```diff
@@ -27,11 +28,13 @@ def __init__(self,
         super().__init__(continuous=True)
 
         self._goalside_ratio = goalside_ratio  # Ratio of angle defending goal line versus attacker line
-        self._mark_robot = mark_robot  # Robot we are defending against
+        # Robot we are defending against
+        self._mark_robot = eval_opp.get_closest_opponent(robocup.Point(0, 0)) \
```

Comment:
This is weird. Curious how does this work?

Reply:
If we don't assign a robot, then the wing defender marks the closest robot to our goal